lib/membrane_http_adaptive_stream/stream_sink_bin.ex

```diff
@@ -0,0 +1,139 @@
+defmodule Membrane.HTTPAdaptiveStream.StreamSinkBin do
+  @moduledoc """
+  This bin takes a parsed video or a parsed audio stream or both and produces an HLS playlist.
+
+  The bin payloads and muxes the incoming streams to the CMAF format and dumps an HLS playlist produced from it to storage.
```

Comment:
HLS is one possibility, in the future we could support different http streams and this bin won't change. You can delete this line if you apply the previous comment.